python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1919,3 +1919,11 @@ def test_is_cudf_pandas():
 
         assert not isinstance_cudf_pandas(obj._fsproxy_slow, np.ndarray)
         assert not isinstance_cudf_pandas(obj._fsproxy_fast, np.ndarray)
+
+
+def test_series_dtype_property():
+    s = pd.Series([1, 2, 3])
+    xs = xpd.Series([1, 2, 3])
+    expected = np.dtype(s)
```

Comment:
Wow I never seen someone use `np.dtype` on a pandas object but TIL it works

Reply:
Yeah I saw it in the failing integration tests. I think its called somewhere in `tensorflow`